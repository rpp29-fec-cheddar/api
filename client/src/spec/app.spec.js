/*eslint-env es6*/
/* eslint-disable no-unused-vars */
/* eslint-disable no-eval */
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../app.jsx';
import 'regenerator-runtime/runtime';

// eslint-disable-next-line no-undef
describe('App', () => {
  // eslint-disable-next-line no-undef
  test('Test that App renders', () => {
    const component = renderer.create(<App ></App>);
    let tree = component.toJSON();
    // eslint-disable-next-line no-undef
    expect(tree).toMatchSnapshot();
  });
});


// eslint-disable-next-line no-undef
describe('true is truthy and false is falsy', () => {
  // eslint-disable-next-line no-undef
  test('true is truthy', () => {
    // eslint-disable-next-line no-undef
    expect(true).toBe(true);
  });

  // eslint-disable-next-line no-undef
  test('false is falsy', () => {
    // eslint-disable-next-line no-undef
    expect(false).toBe(false);
  });
});