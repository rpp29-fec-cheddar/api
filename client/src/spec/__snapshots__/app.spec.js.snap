// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test that App renders 1`] = `
<div>
  <h1 />
  <div>
    <div>
      Overview Here!
    </div>
  </div>
  <br />
  <div>
    <h2>
      Related Products
    </h2>
    <div
      className="productCard"
    />
    <h2
      className="yourOutfitTitle"
    >
      Your Outfit
    </h2>
    <div
      className="yourOutfitCard"
    >
      <div
        className="img"
      >
        X
      </div>
      <div
        className="cardCategory"
      >
        Category
      </div>
      <div
        className="expProductName"
      >
        Expanded product name with extra text
      </div>
      <div
        className="cardPrice"
      >
        $80.00
      </div>
      <div
        className="cardRating"
      >
        *****
      </div>
    </div>
  </div>
  <br />
  <br />
  <div>
    <div>
      <h1>
        0
      </h1>
      <div>
        <div
          className="single-star-container"
        >
          <div
            className="single-star-fill"
            style={
              Object {
                "width": "0px",
              }
            }
          >
            <img
              alt="stars alt"
              className="single-star-outline"
              src="star.png"
            />
          </div>
        </div>
        <div
          className="single-star-container"
        >
          <div
            className="single-star-fill"
            style={
              Object {
                "width": "0px",
              }
            }
          >
            <img
              alt="stars alt"
              className="single-star-outline"
              src="star.png"
            />
          </div>
        </div>
        <div
          className="single-star-container"
        >
          <div
            className="single-star-fill"
            style={
              Object {
                "width": "0px",
              }
            }
          >
            <img
              alt="stars alt"
              className="single-star-outline"
              src="star.png"
            />
          </div>
        </div>
        <div
          className="single-star-container"
        >
          <div
            className="single-star-fill"
            style={
              Object {
                "width": "0px",
              }
            }
          >
            <img
              alt="stars alt"
              className="single-star-outline"
              src="star.png"
            />
          </div>
        </div>
        <div
          className="single-star-container"
        >
          <div
            className="single-star-fill"
            style={
              Object {
                "width": "0px",
              }
            }
          >
            <img
              alt="stars alt"
              className="single-star-outline"
              src="star.png"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
